charts/pvc-autoresizer/templates/controller/deployment.yaml

```
@@ -14,6 +14,10 @@ spec:
     metadata:
       labels:
         app.kubernetes.io/name: pvc-autoresizer
+      {{ if .Values.controller.podAnnotations -}}
```

```suggestion
      {{- if .Values.controller.podAnnotations}}
```

Just for consistency.